content/docs/interactive/go.md

```diff
@@ -0,0 +1,194 @@
+---
+
+Interactive examples with Go
+
+### Setting things up
+
+Before we start using the appbase-js lib, we’ll need to set up an appbase.io app.
+
+For this tutorial, we have used a <a href="https://opensource.appbase.io/dejavu/live/#?input_state=XQAAAAKLAQAAAAAAAAA9iIqnY-B2BnTZGEQz6wkFsW71dAzA7YYc-SS2NBdZOu2iiqUDTwzb8SRY-P60qxz_ZFKoJgMwEJushaRl-FxMxQqDCBLVG-xBlA5HfOZXDzUuGnntd_Zw9u4C0YdVJQ8HvMJrVO8AfQy73d9wq7TjySsVRv-NAKU5ZUw4jxU0ynrQflgPkDLN6AGDv4jeOi8Ir9BBSZ-bdv4J2oq7eCzLoC-gl9qTZsTRLHsXPhHvClG5we6nqctwdPgHqEWqj25nG0qo1RkmJYY_ZTF4XEJcMQyIw-2Rck0OE-ZTR7g8d3ste2uR2u9JbeJj9fjtjVNDltaQGN8jaAdUVVriYpB2CzgXN__Rv9tA&editable=false" target="_blank">sample app</a> containing some dummy housing records. All the examples use this app. You can also clone this app (to create your own private app) directly by clicking "Clone this app" in the <a href="https://opensource.appbase.io/dejavu/live/#?input_state=XQAAAAKLAQAAAAAAAAA9iIqnY-B2BnTZGEQz6wkFsW71dAzA7YYc-SS2NBdZOu2iiqUDTwzb8SRY-P60qxz_ZFKoJgMwEJushaRl-FxMxQqDCBLVG-xBlA5HfOZXDzUuGnntd_Zw9u4C0YdVJQ8HvMJrVO8AfQy73d9wq7TjySsVRv-NAKU5ZUw4jxU0ynrQflgPkDLN6AGDv4jeOi8Ir9BBSZ-bdv4J2oq7eCzLoC-gl9qTZsTRLHsXPhHvClG5we6nqctwdPgHqEWqj25nG0qo1RkmJYY_ZTF4XEJcMQyIw-2Rck0OE-ZTR7g8d3ste2uR2u9JbeJj9fjtjVNDltaQGN8jaAdUVVriYpB2CzgXN__Rv9tA&editable=false" target="_blank">data browser view of the app here</a>.
+
+<br/>
+
+<a target="_blank" href="https://opensource.appbase.io/dejavu/live/#?input_state=XQAAAAKLAQAAAAAAAAA9iIqnY-B2BnTZGEQz6wkFsW71dAzA7YYc-SS2NBdZOu2iiqUDTwzb8SRY-P60qxz_ZFKoJgMwEJushaRl-FxMxQqDCBLVG-xBlA5HfOZXDzUuGnntd_Zw9u4C0YdVJQ8HvMJrVO8AfQy73d9wq7TjySsVRv-NAKU5ZUw4jxU0ynrQflgPkDLN6AGDv4jeOi8Ir9BBSZ-bdv4J2oq7eCzLoC-gl9qTZsTRLHsXPhHvClG5we6nqctwdPgHqEWqj25nG0qo1RkmJYY_ZTF4XEJcMQyIw-2Rck0OE-ZTR7g8d3ste2uR2u9JbeJj9fjtjVNDltaQGN8jaAdUVVriYpB2CzgXN__Rv9tA&editable=false">
+<img alt="Dataset" src="https://i.imgur.com/erNycvVg.png" />
+</a>
+
+All the records are structured in the following format:
+
+```js
+ 
+{
+    name: 'The White House',
+    room_type: 'Private room',
+    property_type: 'Townhouse',
+    price: 124,
+    has_availability: true,
+    accommodates: 2,
+    bedrooms: 1,
+    bathrooms: 1.5,
+    beds: 1,
+    bed_type: 'Real Bed',
+    host_image: 'https://host_image.link',
+    host_name: 'Alyson',
+    image: 'https://image.link',
+    listing_url: 'https://www.airbnb.com/rooms/6644628',
+    location: {
+        lat: 47.53540733743967,
+        lon: -122.27983057017123
+    },
+    date_from: 20170426,
+    date_to: 20170421,
+}
+```
+
+To interact with this dataset, we'll make HTTP REST API requests. The semantics should remain the same for any 3rd party HTTP library as well.
+
+This is the common code which exists in all examples. Here, we simply create `url` based on [`app`](/concepts/datamodel.html#app-span-stylefont-weight-200aka-indexspan), [`type`](/concepts/datamodel.html#type), a `payload` containing request body and set headers with `credentials`.
```

Comment:
We might want to rephrase this a little.
In JavaScript, we [changed it](https://github.com/appbaseio/Docs/commit/98f00dfc93321ea7780cd123e64790f18ac1a6f3#diff-3e09e98e90115ae9883a737361c651c4R55) to `...import the lib and create a reference object...` which might not fit here.